main/templates/bit/tagmanager.html

```diff
@@ -0,0 +1,11 @@
+# if not current_user.admin and config.CONFIG_DB.tagmanager_id
+    <!-- Google Tag Manager -->
```

Comment:
- 2 spaces for indentation
- We could avoid the comments.. as it's obvious from the domain I would say :)